code/game/machinery/machinery.dm

```diff
@@ -396,15 +396,18 @@ Class Procs:
 /obj/machinery/proc/default_deconstruction_crowbar(var/obj/item/weapon/crowbar/C, var/ignore_panel = 0)
 	if(istype(C) && (panel_open || ignore_panel))
 		playsound(loc, C.usesound, 50, 1)
-		spawn_frame()
-		for(var/obj/item/I in component_parts)
-			if(I.reliability != 100 && crit_fail)
-				I.crit_fail = 1
-			I.forceMove(loc)
-		qdel(src)
+		deconstruct()
 		return 1
 	return 0
 
+/obj/machinery/proc/deconstruct(disassembled = TRUE)

```

Comment:
It is https://github.com/tgstation/tgstation/blob/7a53521b1d728c3c480244f861568fb8772dcdf5/code/game/objects/obj_defense.dm#L223